src/main/cliapp/src/containers/genesPage/GenesTable.js

```diff
@@ -27,6 +27,18 @@ export const GenesTable = () => {
 		}
 	};
 
+	const symbolBodyTemplate = (rowData) => {
+		if (rowData?.geneFullName) {
+			return (
+				<>
+					<EllipsisTableCell otherClasses={`a${rowData.curie.replace(':', '')}`}>
+						{rowData.geneFullName.displayText}
```

Comment:
I think it would be better to do it at this level?

Reply:
But then we will have to do it everywhere like in all templates.

Reply:
The ticket is for Gene symbol & name columns. Where else will this change needs to be added? I change it at individual table level as suggested by you.

Reply:
Alleles, DA table is already taken care of . So, I have changed it in Genes page only.

Reply:
There is a reason why its called: "dangerouslySetInnerHTML" we want to make sure and use only when we know what the text is thats got HTML in it.